app/controllers/api/v0/followers_controller.rb

```diff
@@ -24,6 +24,12 @@ def limit_per_page(default:, max:)
         per_page = (params[:per_page] || default).to_i
         @follows_limit = [per_page, max].min
       end
+
+      def sort_param
+        return :asc if params[:sort] == "asc"
```

Comment:
Looking at it from the perspective of a consumer of this API, what does it mean to sort followers in "ascending" order?

Reply:
This ties in with @rhymes comment above re sort params and the future direction of sorting in the API: https://github.com/forem/forem/pull/13287#pullrequestreview-629799466